sonar-orchestrator/src/main/java/com/sonar/orchestrator/OrchestratorBuilder.java

```diff
@@ -165,6 +165,23 @@ public OrchestratorBuilder setServerProperty(String key, @Nullable String value)
     return this;
   }
 
+  /**
+   * Enable JDWP agent in the CE process for remote debugging on port 5006
+   */
+  public OrchestratorBuilder enableCeDebug() {
+    this.setServerProperty("sonar.ce.javaAdditionalOpts", "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5006");
```

Comment:
Just a thought, maybe it would be worth to fail on this method if we detect that we are on Cirrus CI?

Reply:
Good idea!